addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -29,6 +29,7 @@ class ChannelMember(models.Model):
     custom_channel_name = fields.Char('Custom channel name')
     fetched_message_id = fields.Many2one('mail.message', string='Last Fetched', index="btree_not_null")
     seen_message_id = fields.Many2one('mail.message', string='Last Seen', index="btree_not_null")
+    new_message_separator_position = fields.Integer(string='Position of the separator (id of the message)', default=0)
```

Comment:
I'm not sure if the extra verbosity of `_position` adds much value to the field name.

Also it would be nice to specify if the separator is before or after the message with that ID.

Probably better to explain in help rather than in string, which should be simpler.

Also would be better with double quote.